### PR TITLE
Add workflow to close stale issues and PRs

### LINKED ISSUE #491: Introduce stale bot
There are many issues from past that has not been answered. I can introduce stale bot with the similar configuration of etcd repo.
https://github.com/etcd-io/etcd/blob/main/.github/stale.yml

Any comments?

### PR DESCRIPTION
**Pull request by cenkalti**
Fixes #491

### CODE REVIEW

File: .github/workflows/stale.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-stale: 60
+          days-before-close: 7
```
Comment (ahrtr) [inline .github/workflows/stale.yaml:17]: Let's keep the values consistent with etcd.
```suggestion
          days-before-stale: 90
          days-before-close: 21
```

**Review by ahrtr — COMMENTED** [review]
Thanks for the PR. Only a minor comment.

**Comment by ahrtr** [comment]
@cenkalti could you please signoff the commit?

**Comment by cenkalti** [comment]
Done.

**Review by ahrtr — APPROVED** [review]
LGTM

Thank you

**Pull request merged**